services/workflowbuilder/src/infra/database/prisma/repositories/prisma-procedureProject-repository.ts

```diff
@@ -17,4 +19,40 @@ export class prismaProcedureProjectRepository
       data: procedureProjectPrismaData,
     });
   }
+
+  async findByField(
+    field: string,
```

Comment:
Can you discriminate which fields, by that we already have?
```suggestion
    field: 'id' | 'machineName' | 'name',
```

In order to only find by the searchable fields